### ISSUE TITLE
Question about tsdf

### ISSUE THREAD

**Issue by PlayerUsr** [andyzeng/tsdf-fusion#2]
Hi,

Is what I can calculate tsdf for a cloud of points not for depth images ?
You can help me with an idea 

Thank you for your help.

**Comment by andyzeng** [comment]
Hello, it is possible to compute TSDF from a point cloud. However, to make use of negative distance values, you'll need to know "occluded" space in your point cloud. This repository currently does not have support for that.

But for what it's worth, you can still compute TDF, which is the unsigned version of the TSDF. You can find code to compute TDF from point clouds [here](https://github.com/andyzeng/3dmatch-toolbox#converting-3d-data-to-tdf-voxel-grids).

**Comment by PlayerUsr** [comment]
Thank you very much.

Is it possible to use it without Marvin (not working with windows) Just for compute TDF voxel grid values given a point cloud.
Thank you in advance.

**Comment by andyzeng** [comment]
Yes, it is possible to use it without Marvin. All of the code [here](https://github.com/andyzeng/3dmatch-toolbox#converting-3d-data-to-tdf-voxel-grids) for computing TDF does not require Marvin. For [demo.cu](https://github.com/andyzeng/3dmatch-toolbox/blob/master/core/demo.cu), you can remove the inference parts that use Marvin.

**Comment by PlayerUsr** [comment]
Thanks a lot sir for your support with your useful articles All right , i have one more question
I want to inform you that adapted your algorithm on my (nuage de points) to mesure my TDF and the values between 0 and 1
In the other hand i wonder if i can visualize the results (like tsdf) with a .ply format
Or with matlab
Returning to the negative distance values , u mentioned that i should know the occluded space in point cloud
Could u please clarify this point to me or could u join a link that explines it
Thanks for your permenant support.
Thanks so muck.

**Comment by PlayerUsr** [comment]
Thanks a lot sir for your support with your useful articles All right , i have one more question I want to inform you that adapted your algorithm on my (nuage de points) to mesure my TDF and the values between 0 and 1.

1- In the other hand i wonder if i can visualize the results (like tsdf) with a .ply format
Or with matlab.

2- Returning to the negative distance values , u mentioned that i should know the occluded space in point cloud.
Could u please clarify this point to me or could u join a link that explines it
Thanks for your permenant support
Thanks you so much.

**Comment by PlayerUsr** [comment]
Hi @andyzeng 

Please Returning to the negative distance values , u mentioned that i should know the occluded space in point cloud.
Could u please clarify this point to with adea.
Thank you in advance.
